scripts/deploy.sh

```diff
@@ -142,10 +152,10 @@ fi
 cd ${RPCD_DIR}/playbooks/
 
 # build the RPC python package repository
-run_ansible repo-build.yml
+#run_ansible repo-build.yml
```

Comment:
@pronetheanfire yeah, we can also remove the plays as we're not using them any more. Again, was trying to keep the patch size down but I guess in this case it should be done.